### ISSUE TITLE
Remove use of int pin from simpletest example?

### ISSUE THREAD

**Issue by caternuson** [adafruit/Adafruit_CircuitPython_LIS3DH#64]
Make simpletest even more simple. The int pin is not really needed for basic accelo readings.

**Comment by ladyada** [comment]
thats fine, its needed for the tap examples

**Comment by caternuson** [comment]
yep. totes. and looks like there's a good one for that.

this would be specific to just the `lis3dh_simpletest.py ` example. something super simple.
```python
Adafruit CircuitPython 5.3.1 on 2020-07-13; Adafruit ItsyBitsy M4 Express with samd51g19
>>> import board
>>> import adafruit_lis3dh
>>> lis = adafruit_lis3dh.LIS3DH_I2C(board.I2C())
>>> lis.acceleration
acceleration(x=0.325669, y=0.354405, z=9.93291)
>>> 
```
i'll pr something....

**Comment by caternuson** [comment]
Fixed with #65
